### PR TITLE
New Generative Anyscale Module in 1.23!

### PR DESCRIPTION
**Pull request by CShorten**
Added documentation on how to configure, supported models, and other info.

### Type of change:

- [X] **Documentation** updates (non-breaking change to fix/update documentation)
- [ ] **Website** updates (non-breaking change to update main page, company pages, pricing, etc)
- [ ] **Content** updates – **blog**, **podcast** (non-breaking change to add/update content)
- [ ] **Bug fix** (non-breaking change to fixes an issue with the site)
- [ ] **Feature** or **enhancements** (non-breaking change to add functionality)

### How Has This Been Tested?

- [ ] **Github action** – automated build completed without errors
- [X] **Local build** - the site works as expected when running `yarn start`

> note, you can run `yarn verify-links` to test site links locally

### CODE REVIEW

File: developers/weaviate/modules/reader-generator-modules/generative-anyscale.md [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+      ]
+    }
+  }
+}
+```
+
+#### Example - grouped result
+
+Here is an example of a query where:
+* we run a vector search (with `nearText`) to find publications about finance,
```
Comment (daveatweaviate) [inline developers/weaviate/modules/reader-generator-modules/generative-anyscale.md:253]: If you update the example above, please update this one to match

**Review by daveatweaviate — COMMENTED** [review]
@CShorten This is a huge help, thanks! 

There's a typo and a couple of small suggestions for format, but this is great.

File: developers/weaviate/modules/reader-generator-modules/generative-anyscale.md [hunk 2]
```diff
@@ -216,30 +216,30 @@ You can add both `title` and `summary` to the prompt by enclosing them in curly
 #### Example - single result
 
 Here is an example of a query where:
-* we run a vector search (with `nearText`) to find articles about "Italian food"
-* then we ask the generator module to describe each result as a Facebook ad.
-  * the query asks for the `summary` field, which it then includes in the `prompt` argument of the `generate` operator.
+* we get a podcast clip (with limit 1)
```
Comment (daveatweaviate) [inline developers/weaviate/modules/reader-generator-modules/generative-anyscale.md:219]: Consider parallel list structure: 

* We get a podcast clip (with limit 1).
* We ask the generator module to summarize the content into one sentence.
* The query

File: developers/weaviate/modules/reader-generator-modules/generative-anyscale.md [hunk 3]
```diff
@@ -216,30 +216,30 @@ You can add both `title` and `summary` to the prompt by enclosing them in curly
...
+* then we ask the generator module to summarize the content into one sentence.
+  * the query asks for the `speaker` and `content` fields, which are then included in the `prompt` argument of the `generate` operator.
```
Comment (daveatweaviate) [inline developers/weaviate/modules/reader-generator-modules/generative-anyscale.md:221]: Consider

`content` fields. It uses them in the `prompt`

File: developers/weaviate/modules/reader-generator-modules/generative-anyscale.md [hunk 4]
```diff
@@ -250,40 +250,57 @@ import AWSSingleResult from '/_includes/code/generative.aws.singleresult.mdx';
 #### Example - grouped result
 
 Here is an example of a query where:
-* we run a vector search (with `nearText`) to find publications about finance,
-* then we ask the generator module to explain why these articles are about finance.
+* we run a vector search (with `nearText`) to find podcast clips semantically similar to `"What is ref2vec?"`
```
Comment (daveatweaviate) [inline developers/weaviate/modules/reader-generator-modules/generative-anyscale.md:253]: Consider parallel list structure:

* We run ... ref2vec?"`.
* We ask ... results.

**Review by daveatweaviate — COMMENTED** [review]
@CShorten  looking good, thanks!

File: developers/weaviate/modules/reader-generator-modules/generative-anyscale.md (hunk 4 above)
Comment (databyjp) [reply developers/weaviate/modules/reader-generator-modules/generative-anyscale.md:253]: Hey @ this is more or less verbatim the same as the other `generative` module examples. So I'll see if we can refactor all of them together.

File: developers/weaviate/modules/reader-generator-modules/generative-anyscale.md (hunk 3 above)
Comment (databyjp) [reply developers/weaviate/modules/reader-generator-modules/generative-anyscale.md:221]: Closing for now for the same reason as below.

File: developers/weaviate/modules/reader-generator-modules/generative-anyscale.md (hunk 2 above)
Comment (databyjp) [reply developers/weaviate/modules/reader-generator-modules/generative-anyscale.md:219]: Closing for now for the same reason as below.

File: developers/weaviate/modules/reader-generator-modules/generative-anyscale.md [hunk 5]
```diff
@@ -0,0 +1,301 @@
...
+        }
+      }
+    }
+  }
+}
+```
+
+#### Example - single result
+
+Here is an example of a query where:
```
[the remark this answers is not in the document]
Comment (databyjp) [reply developers/weaviate/modules/reader-generator-modules/generative-anyscale.md:218]: Closing for now for the same reason as below.

**Pull request merged**